vars.tf

```diff
@@ -127,6 +127,31 @@ variable "zone" {
 
 
 
+# GKE related variables
```

Comment:
First of all please add a comment in the file with a link to optional versions.
Secondly, take a look at this data source:
https://registry.terraform.io/providers/hashicorp/google/latest/docs/data-sources/container_engine_versions

Reply:
Added